Src/BootCamp.Chapter/Shop.cs

```diff
@@ -64,6 +75,17 @@ public decimal Buy(Item item)
         /// </returns>
         public Item Sell(string item)
         {
+            //TODO Sell- adds money to the shop. Shop can only sell items that it has. 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Complete the task associated to this 'TODO' comment.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5148522)

Reply:
fixed this in my new local build will upload as soon as reviewed.